Cargo.toml

```diff
@@ -30,6 +30,12 @@ petgraph = { version ="0.6", optional = true }
 pp-rs = { version = "0.2.1", optional = true }
 hexf-parse = { version = "0.2.1", optional = true }
 
+[profile.release]
+panic = "abort"
+
+[profile.dev]
```

Comment:
I don't think this is a decision for the library to make, applying it to the `cli` makes more sense, but even then I find the backtraces helpful when debugging panics.

Reply:
Having this in the Cargo.toml doesn't affect users of this as a library. i.e. only the top level `[profile]` section applies.

I'm also pretty sure this doesn't impact your ability to get backtraces when debugging panics.

Reply:
Yeah, the backtrace/printing is done by the panic hook, which always fires.

Reply:
Oh I didn't know only the top level profile counted.

Don't we lose stack unwinding if we change the abort strategy to abort?

Reply:
Oh wait seems like `abort` does have backtraces what I was thinking about is an unstable feature called `panic_immediate_abort`

Reply:
> Having this in the Cargo.toml doesn't affect users of this as a library

Who does it affect then? Are we improving the developer velocity here?

Reply:
I guess we'll just have slightly better compile times and exec size for naga-cli